src/AddIns/Uno.UI.MediaPlayer.Skia.Gtk/GtkMediaPlayer.cs

```diff
@@ -227,13 +221,7 @@ public void Pause()
 				this.Log().Debug("Pause");
 			}
 
-			_ = Task.Run(() =>
-			{
-				// Required to be on a background thread
-				// to avoid deadlocks
-				_mediaPlayer.Pause();
-			});
-
+			_mediaPlayer.Pause();
 			_videoView.SetVisible(true);
```

Comment:
@jeromelaban Why is making the view visible here necessary? Is there a a scenario where the view isn't already visible when pausing? And in that scenario, would we want to unhide the view?

Reply:
I don't recall. Let's merge as-is and we'll advise.